java-runtime-builder-app/src/test/resources/testWorkspaces/customBuildWorkspace/app.yaml

```diff
@@ -0,0 +1,4 @@
+env: flex
+runtime: java
+runtime_config:
+  build_script: "bazel build //:all"
```

Comment:
I'm going to take care of this in a separate PR that deals with test refactoring more generally.